pkg/endpoint/local_ip.go

```diff
@@ -40,12 +40,24 @@ func getLocalIPv4ForDestination(dst string) string {
 	return localAddr.IP.String()
 }
 
+func GetLocalIPv6ForDestination(dst string) string {
+	conn, err := net.Dial("udp6", "["+dst+"]:53")
+	logger.FatalOnError(err, "Error getting local IPV6")
+
+	defer conn.Close()
+
+	localAddr := conn.LocalAddr().(*net.UDPAddr)
+
+	return localAddr.IP.String()
+}
+
 func GetLocalIPForDestination(dst string, family k8snet.IPFamily) string {
 	switch family {
 	case k8snet.IPv4:
 		return getLocalIPv4ForDestination(dst)
 	case k8snet.IPv6:
-		// TODO_IPV6: add V6 healthcheck IP
+		result := GetLocalIPv6ForDestination(dst)
+		return result
```

Comment:
```suggestion
		return GetLocalIPv6ForDestination(dst)
```

to follow the same pattern as IPv4.

Or how about

```
func GetLocalIPForDestination(dst string, family k8snet.IPFamily) string {
	conn, err := net.Dial("udp" + family, "["+dst+"]:53")
	logger.FatalOnError(err, "Error getting local IPv" + family)

	defer conn.Close()

	return conn.LocalAddr().(*net.UDPAddr).IP.String()
}
```